examples/typescript/main.ts

```diff
@@ -0,0 +1,15 @@
+import { lift, main, std } from "ts4w";
+import { cloud } from "@winglang/sdk";
```

Comment:
Ideally you should use the re-exported cloud namespace to avoid direct interactions with @winglang/sdk (it's a transitive dep)

```suggestion
import { cloud, lift, main, std } from "ts4w";
```